packages/website/docs/state.md

```diff
@@ -52,7 +52,8 @@ You can also manually update the state using setters. It's useful to implement c
 For example, let's say you want to let users fill the search input with the value of a suggestion by clicking or tapping it. You can use the [`setQuery`](state#setquery) setter provided by [`getSources`](sources#getsources) to attach an event when clicking the tap-ahead button and manually set the query.
 
 ```js
-const autocomplete = createAutocomplete({
+autocomplete({
+  // ...
   getSources({ query, setQuery, refresh }) {
     return [
       {
```

Comment:
Not directly related to this PR but we don't support DOM anymore so the example below won't work anymore.

```jsx
autocomplete({
  // ...
  getSources({ setQuery, refresh }) {
    return [
      {
        // ...
        templates: {
          item({ item }) {
            return (
              <div className="ItemContent">
                <div className="aa-aa-ItemSourceIcon">Icon</div>
                <div className="aa-ItemTitle">{item.query}</div>
                <button
                  className="aa-ItemActionButton"
                  onClick={(event) => {
                    event.stopPropagation();

                    setQuery(item.query);
                    refresh();
                  }}
                >
                  Fill query
                </button>
              </div>
            );
          },
        },
      },
    ];
  },
});
```

Reply:
This is JSX, so this does work with adding `h` and turning the snippet into `jsx` no?

```jsx
/** @jsx h */
import { h } from 'preact';
```

Reply:
Yep

Reply:
Handled in a725011.